blues/simulation.py

```diff
@@ -226,35 +215,33 @@ def simulateNCMC(self, verbose=False, write_ncmc=False):
         """Function that performs the NCMC simulation."""
         #append nc reporter at the first step
         if (self.current_iter == 0) and (write_ncmc):
-            self.ncmc_reporter = app.dcdreporter.DCDReporter(self.ncmc_outfile, 1) 
+            self.ncmc_reporter = app.dcdreporter.DCDReporter(self.ncmc_outfile, 1)
             self.nc_sim.reporters.append(self.ncmc_reporter)
         for nc_step in range(self.nstepsNC):
             try:
                 self.current_stepNC = int(nc_step)
                 # Calculate Work/Energies Before Step
-                work_initial = self.getWorkInfo(self.nc_integrator, self.work_keys)
-                     
+                if verbose:
+                    work_initial = self.getWorkInfo(self.nc_integrator, self.work_keys)
 
                 # Attempt NCMC Move
-                if int(self.mover.moves['step']) == nc_step:
-                    print('[Iter {}] Performing NCMC {} move'.format(
-                    self.current_iter, self.mover.moves['method'].__name__))
+                if self.nstepsNC / 2 == nc_step:
```

Comment:
This is the portion that specifies that the move is done halfway through the NCMC protocol (so that it's symmetric). I can add a comment to clarify that.